app/posts/manage-breast-screening/2025/05/2025-05-06-image-reading-first-prototype.md

```diff
@@ -0,0 +1,112 @@
+---
+  title: Image reading - first prototype
+  date: 2025-05-06
+  tags:
+    - alpha
+    - prototype
+    - image reading
+---
+
+Building on the [previous design explorations](/manage-breast-screening/2025/04/existing-image-reading-design-concepts/) of breast image reading, our initial image reading prototype focuses on three key user needs for image readers: choosing which cases to read, giving opinions on cases and efficiently navigating between them. We've deferred addressing first and second read communication for future design iterations.
```

Comment:
"first and second read communication" is a bit confusing to me? I can have a guess what it means, but is there any way to briefly reword this for clarity?